app/templates/breadcrumb.html

```diff
@@ -1,5 +1,9 @@
 <ol>
-  <% _.each(parents, function iterator(parent, key) { %><li>
-      <a href="#<%- parent.url %>"><%- parent.title %></a></li>
```

Comment:
Yes, but if it in the template, other projects which uses gohan_webui (like cloudwan_webui) needs to escape by itself, it tends to be forgotten too. My idea is it might be better to handle it in view to avoid it. What do you think?